src/core/Piano.cpp

```diff
@@ -67,9 +67,9 @@ Piano::Piano( InstrumentTrack* track ) :
 	m_instrumentTrack( track ),
 	m_midiEvProc( track )        /*!< the InstrumentTrack Model */
 {
-	for( int i = 0; i < NumKeys; ++i )
+	for (bool& pressedKey : m_pressedKeys)
```

Comment:
Another case for `std::array` and `fill`

Reply:
Could probably simplify this out completely with value initialization though.